src/main/java/com/mparticle/kits/AppsFlyerKit.java

```diff
@@ -54,9 +55,13 @@ public String getName() {
         AppsFlyerLib.getInstance().startTracking((Application) context.getApplicationContext(), getSettings().get(DEV_KEY));
         AppsFlyerLib.getInstance().setDebugLog(MParticle.getInstance().getEnvironment() == MParticle.Environment.Development);
         AppsFlyerLib.getInstance().setCollectAndroidID(MParticle.isAndroidIdDisabled() == false);
+        HashMap<String, String> integrationAttributes = new HashMap<String, String>(1);
+        integrationAttributes.put(APPSFLYERID_INTEGRATION_KEY, AppsFlyerLib.getInstance().getAppsFlyerUID(context));
```

Comment:
this looks great - the only question/concern I have, though, is whether the UID will ever change during a session. Can you ask AppsFlyer if it ever changes?